nautobot/extras/tests/test_plugins.py

```diff
@@ -485,6 +485,69 @@ def test_relationship_association_validator_raises_exception(self):
             relationship_assoc.clean()
 
 
+@skipIf(
+    "example_plugin" not in settings.PLUGINS,
+    "example_plugin not in settings.PLUGINS",
+)
+class ExampleModelCustomActionViewTest(TestCase):
+    """Test for custom action view `all_names` added to Example App"""
+
+    model = ExampleModel
+
+    @classmethod
+    def setUpTestData(cls):
+        ExampleModel.objects.create(name="Example 1", number=100)
+        ExampleModel.objects.create(name="Example 2", number=200)
+        ExampleModel.objects.create(name="Example 3", number=300)
+        cls.custom_view_url = reverse("plugins:example_plugin:examplemodel_all-names")
```

Comment:
Not a huge fan of the mixed dashes/underscores here... anything we can or should do about that without making a breaking change?

Reply:
Specifying `url_name` on the` @action` header fixed the issue.